kibana-reports/public/components/main/main_utils.tsx

```diff
@@ -59,9 +59,7 @@ export const addReportsTableContent = (data) => {
       sender: `\u2014`,
       kibanaRecipients: `\u2014`,
       emailRecipients: `\u2014`,
-      reportSource: reportParams.report_name.substring(
-        0, reportParams.report_name.lastIndexOf('_') 
-      ), // remove uuid from report name to display report source
+      reportSource: reportParams.description, // display report source name
```

Comment:
Is this a typo? We are assigning description to the source field

Reply:
Not a typo- we need a place to store the raw report source name without parsing out any special characters. And the description field is not used anywhere else.